cloudformation/template.yml

```diff
@@ -0,0 +1,339 @@
+
+  BatchServiceRolePolicyARN:
+    Type: String
+    Default: 'arn:aws:iam::aws:policy/service-role/AWSBatchServiceRole'
+    Description: The ARN of the IAM role to assign to running Batch instances
+
+  SpotFleetServiceRolePolicyARN:
+    Type: String
+    Default: 'arn:aws:iam::aws:policy/service-role/AmazonEC2SpotFleetTaggingRole'
+    Description: The ARN of the IAM role to assign to running spot fleet instances
+
+  RepositoryName:
+    Type: String
+    Default: 'raster-vision-testing'
+    Description: Specifies the name of the ECR repository to create and retrieve images from
+
+  ImageName:
+    Type: String
+    Default: 'raster-vision-gpu'
+    Description: Specifies the name of the container image to pull for Batch jobs
+
+  ImageTag:
+    Type: String
+    Default: latest
+    Description: Tag of the container image to retrieve from ECR
+
+  VPC:
+    Type: 'AWS::EC2::VPC::Id'
+    Description: Virtual Private Cloud in which to launch Batch instances
+
+  SubnetIds:
+    Type: 'List<AWS::EC2::Subnet::Id>'
+    Description: >
+      A list of IDs of subnets in which to launch Batch instances (all subnets
+      must exist in the VPC you selected)
+
+Resources:
+  BatchServiceIAMRole:
+    Type: 'AWS::IAM::Role'
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          -
+            Effect: Allow
+            Action:
+              - 'sts:AssumeRole'
+            Principal:
+              Service:
+                - batch.amazonaws.com
+      ManagedPolicyArns:
+        - !Ref BatchServiceRolePolicyARN
+
+  SpotFleetIAMRole:
+    Type: 'AWS::IAM::Role'
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          -
+            Effect: Allow
+            Action:
+              - 'sts:AssumeRole'
+            Principal:
+              Service:
+                - spotfleet.amazonaws.com
+      ManagedPolicyArns:
+        - !Ref SpotFleetServiceRolePolicyARN
+
+  BatchInstanceIAMRole:
+    Type: 'AWS::IAM::Role'
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          -
+            Effect: Allow
+            Action:
+              - 'sts:AssumeRole'
+            Principal:
+              Service:
+                - ec2.amazonaws.com
+      ManagedPolicyArns:
+        - 'arn:aws:iam::aws:policy/service-role/AmazonEC2ContainerServiceforEC2Role'
+        - 'arn:aws:iam::aws:policy/AmazonS3FullAccess'
+
+  BatchInstanceProfile:
+    Type: 'AWS::IAM::InstanceProfile'
+    Properties:
+      Path: /
+      Roles:
+        - !Ref BatchInstanceIAMRole
+      InstanceProfileName: !Join ['', ['BatchIAM', !Ref Environment]]
+
+  ContainerInstanceSecurityGroup:
+    Type: 'AWS::EC2::SecurityGroup'
+    Properties:
+      VpcId: !Ref VPC
+      GroupDescription: !Join ['', ['Security group for ', !Ref Project, ' (', !Ref Environment, ')']]
```

Comment:
`GroupDescription` is a [required attribute](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-security-group.html#cfn-ec2-securitygroup-groupdescription) in CloudFormation.

Reply:
Yeah, I think that Terraform defaults it to something like `Managed by Terraform`.